### PR TITLE
Fix #1148 - String import for unsupported locales

### LINKED ISSUE #1148: String import/export: cak / trs / zam / mix  / meh is not a valid locale
Output from Android2Po when importing/exporting strings:

```
    [failed] cak is not a valid locale
    [failed] trs is not a valid locale
    [failed] zam is not a valid locale
```

### PR DESCRIPTION
**Pull request by mastizada**
This is import only. Export is not supported.

Solved using list of missing locales and matched them with locales that has same plural rules with them (using info from pontoon).

### CODE REVIEW

**Comment by mastizada** [comment]
@pocmo Also we can now set custom tags in po files instead of changing them everytime while merging with l10n files from pontoon.

File: tools/l10n/android2po/commands.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 from termcolor import colored
 
 import convert
+from patch import read_po
```
Comment (pocmo) [inline tools/l10n/android2po/commands.py]: Having a patch file is a nice idea!

**Comment by pocmo** [comment]
This works great.

I see some XHTML errors. But they are actually in the translations. I'll file separate issues for that.

**Comment by mastizada** [comment]
Forgot to change message, after our discussion in IRC  I've also added export.

**Pull request merged**